public/app/scss/app.scss

```diff
@@ -1,6 +1,7 @@
-body {
-  width: 100%;
-  padding: 0;
-  margin: 0;
-  font-size: 14px;
-}
+@import './variable.scss';
```

Comment:
Nên đặt rõ ràng cái biến để làm gì ra e.
ví dụ: file dùng để lưu các biến như là color thì đặt là variablesColor.scss để biết mục đích của file này là để làm gì. Như ở đây a đổi tên file thành variablesGlobal.scss có nghĩa là biến toàn cục ấy...